spec/spec_helper_acceptance.rb

```diff
@@ -1,112 +1,59 @@
-require 'beaker-pe'
-require 'beaker-puppet'
-require 'beaker-rspec/spec_helper'
-require 'beaker-rspec/helpers/serverspec'
-require 'sql_testing_helpers'
-require 'beaker/puppet_install_helper'
-require 'beaker/testmode_switcher'
-require 'beaker/testmode_switcher/dsl'
-
-WIN_ISO_ROOT = 'https://artifactory.delivery.puppetlabs.net/artifactory/generic__iso/iso/windows'.freeze
-WIN_2012R2_ISO = 'en_windows_server_2012_r2_with_update_x64_dvd_6052708.iso'.freeze
-QA_RESOURCE_ROOT = 'https://artifactory.delivery.puppetlabs.net/artifactory/generic__iso/iso/SQLServer'.freeze
-SQL_2019_ISO = 'SQLServer2019CTP2.4-x64-ENU.iso'.freeze
-SQL_2017_ISO = 'SQLServer2017-x64-ENU.iso'.freeze
-SQL_2016_ISO = 'en_sql_server_2016_enterprise_with_service_pack_1_x64_dvd_9542382.iso'.freeze
-SQL_2014_ISO = 'SQLServer2014SP3-FullSlipstream-x64-ENU.iso'.freeze
-SQL_2012_ISO = 'SQLServer2012SP1-FullSlipstream-ENU-x64.iso'.freeze
-SQL_ADMIN_USER = 'sa'.freeze
-SQL_ADMIN_PASS = 'Pupp3t1@'.freeze
-
-RSpec.configure do |c|
-  # Readable test descriptions
-  c.formatter = :documentation
-  c.before(:suite) do
-    host = find_only_one('sql_host')
-    base_install(host['sql_version'])
-    # Verify that the version in the host config file is indeed the version on the machine
-    execute_powershell_script_on(host, 'Invoke-Sqlcmd -Query "SELECT @@VERSION;" -QueryTimeout 3') do |result|
-      unless result.stdout.include?(host[:sql_version].to_s)
-        raise "Version in host config #{host[:sql_version]} does not match SQL version #{result}"
-      end
-    end
+# frozen_string_literal: true
+
+require 'serverspec'
+require 'puppet_litmus'
+require 'spec_helper_acceptance_local' if File.file?(File.join(File.dirname(__FILE__), 'spec_helper_acceptance_local.rb'))
+include PuppetLitmus
```

Comment:
I _think_ the best practice now is to use the `configure!` functionality, example below:

```ruby
# frozen_string_literal: true

require 'puppet_litmus'
require 'spec_helper_acceptance_local' if File.file?(File.join(File.dirname(__FILE__), 'spec_helper_acceptance_local.rb'))

PuppetLitmus.configure!
```

- [Source](https://github.com/puppetlabs/puppetlabs-powershell/blob/f2acaee81110641575d3cc2cbc5c690cd7635fd3/spec/spec_helper_acceptance.rb)

Reply:
Updated